packages/fxa-content-server/app/styles/modules/_settings.scss

```diff
@@ -331,15 +331,8 @@ body.settings #stage .settings {
     margin: 5px 0 0 0;
   }
 
-  .settings-unit-title,
-  .settings-unit-subtitle {
-    text-overflow: ellipsis;
```

Comment:
We may want to double check with @ryanfeeley if he's back (or ask Alex/Chelsea who we should ask). I think the idea was that we should have an ellipsis like in this screenshot (stolen from the linked issue) if the text is too long, but your solution might be preferable (looks good to me!). I'll approve but pending UX/Product approval. 👍
![image](https://user-images.githubusercontent.com/13018240/77436474-8cf92a80-6db1-11ea-9961-6ed1711cac8f.png)

Reply:
Sounds good. FWIW my personal preference is to reflow, otherwise you're arbitrarily limiting certain locales. @ryanfeeley what do you think?

Reply:
We do normally fallback to ellipsis but when looking at that example on mobile, it seems like it's less than ideal.

I like your suggestion. In the absence of Ryan, let's just move forward with your suggestion.